prime-router/metadata/json_schema/organizations/organizations.json

```diff
@@ -444,7 +465,8 @@
         "elr-elims",
         "monkeypox",
         "test",
-        "mars-otc-elr"
+        "mars-otc-elr",
+        "mars-otc-elr-onboarding"
```

Comment:
Welp, it looks like IntelliJ changed a bunch of whitespace stuff on me. This is the thing that I actually changed in this file, but it looks like the whitespace was previously inconsistent so I'm gonna leave those changes in.